scripts/github-actions-setup

```diff
@@ -117,6 +118,17 @@ install_cni_plugins() {
     ls -lah "$CNI_DIR"
 }
 
+install_crun() {
+    git clone https://github.com/containers/crun
```

Comment:
we might need to checkout the `v1.15` branch to avoid the regression: https://github.com/containers/crun/issues/1514

Reply:
I'd say we move forward and wait for the fix.